src/foam/nanos/crunch/Capability.js

```diff
@@ -351,8 +354,6 @@ foam.CLASS({
         var prereqs = crunchService.getPrereqs(getId());
         if ( prereqs != null ) {
           for ( var capId : prereqs ) {
-            // QUESTION if the capability is disabled what would be the status of the prereqs chain?
-            if ( ! this.getEnabled() ) continue;
```

Comment:
i think this is still needed, i thought the question was on how the status of a prereq might affect the result of checkPrereqsChainedStatus if the prereq were disabled, which is that it shouldn't affect the result and just be skipped over when iterating through the list of prereqs, so this continue if not enabled should still be checked.

but if you were asking what Capability.getPrereqsChainedStatus should return if the Capability itself is disabled, then I'm not really sure, it probably shouldn't just return GRANTED though and just return the normal result of this method.

Reply:
ooh okok i see now, the old code should have been finding the target capability, yes we want to be checking the prereq.

re:normal result - the check in https://github.com/foam-framework/foam2/pull/4010/commits/ec0b01c80288c3f553b265b7aac531120c918838#diff-7b510a01f4db89bd364afb66eed05ee5R334 shouldn't be necessary because whether or not this capability is enabled doesn't affect the result of its prereqchainedstatus

Reply:
Fixed in https://github.com/foam-framework/foam2/pull/4010/commits/6dede8c2287162db8f7133d62229c77b6ce30c39.